refactor/urls.js

```diff
@@ -1,16 +1,28 @@
 // Refactor this code
 
 function getURL(dev = false, tasks = {}) {
-    let url = dev
-        ? `/tasks?status=ACTIVE&dev=true&size=20`
-        : '/tasks';
+  let url = "/tasks";
 
-    if (tasks.nextTasks) {
-        url += '?hasNext=true';
-    }
+  //   let url = dev ? `/tasks?status=ACTIVE&dev=true&size=20` : "/tasks";
+  let params = new URLSearchParams();
```

Comment:
NIT: Why 'let'?

Reply:
Hmm, good question. 
According to my knowledge, it doesn't make much difference if I use let, const, and var.
but have seen a convention of using let and const among the people, so adhering to it.
Does it make a difference?